src/Symfony/Component/Cache/Adapter/PhpArrayAdapter.php

```diff
@@ -328,7 +328,7 @@ public function commit()
      */
     private function initialize()
     {
-        $this->values = @(include $this->file) ?: array();
+        $this->values = file_exists($this->file) ? (include $this->file ?: array()) : array();
```

Comment:
is_file is slower, and this is a mandatory path. there is no reason this is not a path but in severe mistakes, where the php warning will be enough a report